plugin.video.youtube/resources/lib/youtube_plugin/youtube/helper/subtitles.py

```diff
@@ -172,25 +179,40 @@ def _get(self, language='en', simple_text=None, no_asr=False):
         if (caption_track is None) and has_translation:
             base_url = self.caption_track.get('baseUrl')
             if base_url:
-                subtitle_url = base_url + '&fmt=vtt&type=track&tlang=%s' % language
+                subtitle_url = ''.join([base_url, '&fmt=vtt&type=track&tlang=', language])
```

Comment:
carefull with the vtt format. I am not sure if you are converting it to srt before feeding the player but be aware that in some cases time information is different and the subrip parser that is included with kodi may fail